middleware/cmd/middleware/main.go

```diff
@@ -42,6 +43,7 @@ func main() {
 	argumentMap["bbbCmdScript"] = *bbbCmdScript
 	argumentMap["prometheusURL"] = *prometheusURL
 	argumentMap["redisPort"] = *redisPort
+	argumentMap["firmwareUpdateInfoURL"] = "https://gist.githubusercontent.com/0xB10C/a0e067b7a6e6fdcbdce2fb1f172ec7a5/raw/85136861daa70235e4a843931b123824bb6a69e5/base-icecream.json" //*firmwareUpdateInfoURL
```

Comment:
For review and testing. Will change back to 

```suggestion
	argumentMap["firmwareUpdateInfoURL"] = firmwareUpdateInfoURL
```

before merge.